### DISCUSSION TITLE
cannot connect to server running on pc when I attempt to connect from a device not on the same network ie from my phone when not connected via wifi

### DISCUSSION THREAD

**Discussion by nexusjuan12** [chuot/rdio-scanner#358] [Q&A]
Any insights would be greatly appreciated I assume this is a network issue on my end.

**Comment by cpg178-kcd** [comment]
Make sure the computer you are running RDIO on has the RDIO port open to the internet (port forwarding)

**Comment by doftheworld** [comment]
Go into your router's setup and make sure it is forwarding the port you are using (most likely 3000) to the PC that is running the program. If you are using multiple routers (for instance you cable modem and then another router which your PC is the connected to) you will need to have the cable modem forward the port to the other router and then the other router to the PC.
